passwordResetProcess.php

```diff
@@ -145,6 +147,16 @@
             ]);
 
             if ($mail->Send()) {
+                // Log this password reset request
+                $details = [
+                    'gibbonPersonID' => $gibbonPersonID,
+                    'name' => Format::name('', $row['preferredName'], $row['surname'], 'Staff', false, true),
+                    'changedByID' => $session->get('gibbonPersonID'),
```

Comment:
This page is outside of the normal login session, so the gibbonPersonID and names will need to come from the $row value